docs/source/author/writing_guide.rst

```diff
@@ -292,14 +294,9 @@ Create a Link on a Button (Verto feature)
 Lists
 ==============================================================================
 
-Lists can be created by starting each line with a ``-`` for unordered lists
-or ``1.`` for ordered lists.
-The list needs to be followed by a blank line, however it doesn't require a
-blank line before unless the preceding text is a heading (a blank line is
-then required).
-If you are having issues with a list not rendering correctly, try adding a
-blank line before the list if there is none, otherwise `submit a bug report`_
-if you are still having rendering issues.
+Lists can be created by starting each line with a ``-`` for unordered lists or ``1.`` for ordered lists.
+The list needs to be followed by a blank line, however it doesn't require a blank line before unless the preceding text is a heading (a blank line is then required).
+If you are having issues with a list not rendering correctly, try adding a blank line before the list if there is none, otherwise `submit a bug report`_ if you are still having rendering issues.
 
 .. code-block:: none
 
```

Comment:
> Nested lists can be created by indenting each level by 2 spaces.

Is 4 spaces, we should probably raise that as an issue actually because it's *meant* to be 2 spaces

Reply:
Lets just leave it as 2 in the docs then and fix up existing stuff

Reply:
Will change to 4 spaces